tests/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Properties/VBDefineConstantsEncodingTests.cs

```diff
@@ -0,0 +1,135 @@
+        [InlineData("a=\"b\",c=\"easy\",as=123", "a=/\"b/\",c=/\"easy/\",as=123")]
+        [InlineData("key1=\"value1\",key2=\"value2\",key3=\"value3\"", "key1=/\"value1/\",key2=/\"value2/\",key3=/\"value3/\"")]
+        [InlineData("something=\"equals=this\"", "something=/\"equals/=this/\"")]
+        [InlineData("oh=\"hey=there\",hi=\"didnt=see\",you=\"there\"", "oh=/\"hey/=there/\",hi=/\"didnt/=see/\",you=/\"there/\"")]
+        [InlineData("path=\"/path/to/somewhere/\"", "path=/\"//path//to//somewhere/\"")]
+        [InlineData("file=\"/path/is/here/\"", "file=/\"//path//is//here/\"")]
+        [InlineData("files=\"path=/path/to/somewhere/\"", "files=/\"path/=//path//to//somewhere/\"")]
+        [InlineData("a=1,a=2", "a=1,a=2")]
+        [InlineData("key1  =\"value1  \"", "key1  =/\"value1  /\"")]
+        [InlineData("  key1=\"  value1\"", "  key1=/\"  value1/\"")]
+        [InlineData(" key1 =\" value1 \"", " key1 =/\" value1 /\"")]
+        [InlineData(" =\" \"", " =/\" /\"")]
+        [InlineData("key1=\"a b c d\"", "key1=/\"a b c d/\"")]
+        [InlineData("key1=\"a=b=c=d\"", "key1=/\"a/=b/=c/=d/\"")]
+        [InlineData("key1=\"a=b\"\"c=d\"", "key1=/\"a/=b/\"/\"c/=d/\"")]
+        [InlineData("key1=\"ab\"\"cd\"", "key1=/\"ab/\"/\"cd/\"")]
+        [InlineData("key1=\"\"ab\"\"cd\"\"ef\"", "key1=/\"/\"ab/\"/\"cd/\"/\"ef/\"")]
+        [InlineData("key1=\"\"ab\"\"cd\"", "key1=/\"/\"ab/\"/\"cd/\"")]
+        [InlineData("a b c d=\"value1\"", "a b c d=/\"value1/\"")]
+        [InlineData("a=b=c=d=\"value1\"", "a=b/=c/=d/=/\"value1/\"")]
+        [InlineData("a=\"b\"\"c=d=value1\"", "a=/\"b/\"/\"c/=d/=value1/\"")]
+        [InlineData("\"ab\"\"cd=\"value1\"", "/\"ab/\"/\"cd=/\"value1/\"")]
+        [InlineData("\"\"ab\"\"cd\"\"ef=\"value1\"", "/\"/\"ab/\"/\"cd/\"/\"ef=/\"value1/\"")]
+        [InlineData("\"\"ab\"\"cd=\"value1\"", "/\"/\"ab/\"/\"cd=/\"value1/\"")]
+        [InlineData("\"\"ab\"\"cd=\"     value1     \"", "/\"/\"ab/\"/\"cd=/\"     value1     /\"")]
+        [InlineData("\"\"ab\"\"cd=\"value1     \"", "/\"/\"ab/\"/\"cd=/\"value1     /\"")]
+        [InlineData("\"\"ab\"\"cd=\"     value1\"", "/\"/\"ab/\"/\"cd=/\"     value1/\"")]
+        [InlineData("", "")]
+        [InlineData("val/=ue", "val/=ue")]
+        [InlineData("value","value")]
+        [InlineData("value1,value2,value3", "value1,value2,value3")]
+        [InlineData("value1,value2,value3,key4=\"value4\"", "value1,value2,value3,key4=/\"value4/\"")]
+        [InlineData("value1,key2=\"value2\",value3,value4", "value1,key2=/\"value2/\",value3,value4")]
+        [InlineData("value1,key2=\"value2\",value3,key4=\"value4\"", "value1,key2=/\"value2/\",value3,key4=/\"value4/\"")]
+        [InlineData("key1=\"value1\",value2, value3, value4", "key1=/\"value1/\",value2, value3, value4")]
+        public void ValidVBDefineConstantsEncodingEncoding(string input, string expectedOutput)
```

Comment:
If I understand correctly, this test valides that the encoding is done properly, right? Could we give it a clearer name?